src/NUnitFramework/testdata/TestCaseSourceAttributeFixture.cs

```diff
@@ -60,7 +60,11 @@ private static IEnumerable IgnoredSource
             {
                 return new object[] {
                     new TestCaseData(1),
-                    new TestCaseData(2).Ignore("Don't Run Me!")
+                    new TestCaseData(2).Ignore("Don't Run Me!"),
+                    new TestCaseData(3).Ignore("Ignore Me Until The Future", "4242-01-01"),
+                    new TestCaseData(4).Ignore("Ignore Me Until The Future", "4242-01-01 00:00:00"),
+                    new TestCaseData(5).Ignore("Ignore Me Until The Future", "4242-01-01 00:00:00Z"),
```

Comment:
Should we also have a test where the time is not `00:00:00(Z)`, just to ensure that time is not ignored.